packages/communication-react/review/beta/communication-react.api.md

```diff
@@ -1664,7 +1665,7 @@ export const createStatefulCallClient: (args: StatefulCallClientArgs, options?:
 export const createStatefulChatClient: (args: StatefulChatClientArgs, options?: StatefulChatClientOptions | undefined) => StatefulChatClient;
 
 // @beta (undocumented)
-export const createTeamsCallAdapter: ({ userId, credential, locator }: TeamsCallAdapterArgs) => Promise<TeamsCallAdapter>;
+export const createTeamsCallAdapter: ({ userId, credential, displayName, locator }: TeamsCallAdapterArgs) => Promise<TeamsCallAdapter>;
```

Comment:
from discussion that CTE <--> CTE callees won't see each others' names do we still want to go ahead with this change?